packages/proto-rpc/proto.sh

```diff
@@ -0,0 +1,4 @@
+mkdir -p ./src/proto
+npx protoc --ts_out ./src/proto --ts_opt server_generic,generate_dependencies --proto_path protos protos/ProtoRpc.proto
```

Comment:
Would it be possible add `protoc` as a devDependency and run that dependency directly?

Reply:
protoc is installed with the `@protobuf-ts/plugin` dependency so it wouldn't make too much sense to install it additionally as a devDependency